hibernate-core/src/main/java/org/hibernate/bytecode/enhance/internal/javassist/AttributeTypeDescriptor.java

```diff
@@ -79,6 +83,24 @@ public String buildInLineDirtyCheckingBodyFragment(JavassistEnhancementContext c
 		return builder.toString();
 	}
 
+	private void fetchValeBeforeComparison(JavassistEnhancementContext context, CtClass managedCtClass, CtField currentValue, StringBuilder builder) throws NotFoundException {
+		if ( PersistentAttributesHelper.isAssignable(managedCtClass, PersistentAttributeInterceptable.class.getName() )
+				&& context.isLazyLoadable(currentValue)) {
```

Comment:
There is some code format issue in terms of space char usage. see https://github.com/hibernate/hibernate-ide-codestyles for details or refer to existing code

Reply:
Done.
Please see last commit.